plugins/generator-1.20.4/datapack-1.20.4/jsontriggers/entity_hurt_player.json.ftl

```diff
@@ -0,0 +1,15 @@
+"${registryname}_${cbi}": {
+  "trigger": "minecraft:entity_hurt_player",
+  "conditions": {
+    "damage": {
+      "taken": {
+        "min": ${input$amount_l},
+        "max": ${input$amount_h}
+      },
+      "source_entity": {
+        "type": "${field$source_entity}"

```

Comment:
This will not map correctly as it will use mapping table 0 (entity class names). You need to use mapping table 2 - registry names:

![image](https://github.com/MCreator/MCreator/assets/16374228/55842a1b-f0d5-4daf-b108-e9aed3a15897)

See the example of mapping table selection here:

https://github.com/MCreator/MCreator/blob/d02e808413442aafa37dea6a833795cb0e519b9d/plugins/generator-1.20.4/datapack-1.20.4/templates/biome/biome.json.ftl#L158

I believe this trigger would not work properly with class name here, meaning you should try out the trigger code in the game to double-check if it works, ideally with both vanilla and custom entities.

This comment applies to both MC versions (when commenting on templates, we usually leave comments to one generator but it applies to both versions).